uiServices/src/services.go

```diff
@@ -96,5 +93,50 @@ func validateJsonWithSchema(config []byte) bool {
         }
     }
     return true
-
 }
+
+func getConfigs(rw http.ResponseWriter, req *http.Request){
+
+    configPathDir := req.Header.Get("configPathDir")
+    configName := chi.URLParam(req, "configName")
+
+    if !strings.HasSuffix(configPathDir, "/"){
+        configPathDir = configPathDir + "/"
+    }
+
+    file, err := os.Open(configPathDir + configName + ".xml")
+
+    if len(configPathDir) <= 1{
+        rw.WriteHeader(http.StatusBadRequest)
+        logrus.Println("No Header Path Found")
+        return
+    }
+
+    if err != nil {
+        logrus.Println("Configuration Name Not Found: "+configPathDir + configName)
+        rw.WriteHeader(http.StatusNotFound)
+        return
+    }
+
+    defer file.Close()
+
+    var config perfTestUtils.Config
+
+    byteValue, err := ioutil.ReadAll(file)
+    xml.Unmarshal(byteValue, &config)
+    if err != nil{
+        logrus.Println("Cannot Unmarshall")
+        return
+    }
+
+    configJson, err := json.MarshalIndent(config,"","")
+    if err != nil {
+        logrus.Println("Cannot Marshall")
```

Comment:
logrus.Error
and return a 500. Do not just return -- that is a 200 which means the request was successful.